dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/OAuth20ClientMetatypeService.java

```diff
@@ -236,14 +236,17 @@ private JsonObject createMetatypeJsonForObjectClassDefinition(MetaTypeInformatio
 
     @Trivial
     private boolean isAttributeToIgnore(AttributeDefinition attrDef) {
+        if (attrDef.getID().equals(LibertyOAuth20Provider.KEY_CLIENT_TRUSTED_URI_PREFIXES)) {
+            return false;
+        }
```

Comment:
Removing the `LibertyOAuth20Provider.KEY_CLIENT_TRUSTED_URI_PREFIXES.equals(attributeId)` check in `isUnsupportedRegistrationAttribute()` should make this if statement superfluous.

Reply:
fixed